### DISCUSSION TITLE
Tag or Charge Point Groups

### DISCUSSION THREAD

**Discussion by pinuea** [steve-community/steve#1626] [General]
Hi everybody!

I have SteVe 3.6.0 installed and working with some Charge Points.
I would like to have different types of Tag or Charge Point.
I want some Tags that only work in some Charge Point.

Is this possible?

Thanks for everything!

**Comment by juherr** [comment]
Hi,

The current only way to manage groups with steve is using 1 server instance by group.

**Reply by pinuea** [reply]
Thank you very much for the reply juherr!
Is this planned to be implemented in future versions?

**Reply by juherr** [reply]
Steve is primarily driven by community contributions.

Multi-tenancy will be available if someone make a fork and propose a pull request with it.

But I've no idea if it is a feature that @goekay  will accept.
